core/src/datayoga_core/blocks/redis/lookup/block.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from abc import ABCMeta
+from typing import Any, Dict, List, Optional
+
+import datayoga_core.blocks.redis.utils as redis_utils
+import redis
+from datayoga_core import expression
+from datayoga_core.block import Block as DyBlock
+from datayoga_core.context import Context
+from datayoga_core.result import BlockResult, Result, Status
+from datayoga_core.utils import get_connection_details
+
+logger = logging.getLogger("dy")
+
+
+class Block(DyBlock, metaclass=ABCMeta):
+
+    def init(self, context: Optional[Context] = None):
+        logger.debug(f"Initializing {self.get_block_name()}")
+
+        connection = get_connection_details(self.properties.get("connection"), context)
+        self.redis_client = redis_utils.get_client(
+            connection.get("host"),
+            connection.get("port"),
+            connection.get("password")
```

Comment:
We should also support `user`. If only password is specified, this is the default password. I think that if you pass None as user, it should be the same. Add this to other Redis blocks.

Reply:
✅